pkgs/games/nudoku/default.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, ncurses}:
+let
+  s = 
+  rec {
+    baseName = "nudoku";
+    version = "1.0.0";
+    name = "${baseName}-${version}";
+    url = "https://github.com/jubalh/nudoku/releases/download/${version}/${baseName}-${version}.tar.xz";
+    sha256 = "0nr2j2z07nxk70s8xnmmpzccxicf7kn5mbwby2kg6aq8paarjm8k";
+  };
+  buildInputs = [
+    ncurses
+  ];
+in
+stdenv.mkDerivation {
+  inherit (s) name version;
+  inherit buildInputs;
+  src = fetchurl {
+    inherit (s) url sha256;
+  };
+  preInstall = ''
+    mkdir -p {share/man,bin}
+  '';
+  meta = {
+    inherit (s) version;
+    description = ''A ncurses based sudoku game'';
```

Comment:
`''` are generally used for multi line strings, so regular `"` can be used instead :)